test_scripts/DTLS/common.lua

```diff
@@ -0,0 +1,267 @@
+--! @parameters:
+--! pTbl - policy table to update
+--! @return: none
+--]]
+function m.ptUpdate(pTbl)
+  local filePath = "./files/Security/client_credential.pem"
+  local crt = common.readFile(filePath)
+  pTbl.policy_table.module_config.certificate = crt
+end
+
+--[[ @setSDLConfigParameter: change original value of parameter in SDL .ini file
+--! @parameters:
+--! pParamName - name of the parameter
+--! pParamValue - value to be set
+--! @return: none
+--]]
+function m.setSDLConfigParameter(pParamName, pParamValue)
+  originalValuesInSDL[pParamName] = commonFunctions:read_parameter_from_smart_device_link_ini(pParamName)
+  commonFunctions:write_parameter_to_smart_device_link_ini(pParamName, pParamValue)
+end
+
+--[[ @restoreSDLConfigParameters: restore original values of parameters in SDL .ini file
+--! @parameters: none
+--! @return: none
+--]]
+local function restoreSDLConfigParameters()
+  for pParamName, pParamValue in pairs(originalValuesInSDL) do
+    commonFunctions:write_parameter_to_smart_device_link_ini(pParamName, pParamValue)
+  end
+end
+
+--[[ @postconditions: postcondition steps
+--! @parameters: none
+--! @return: none
+--]]
+function m.postconditions()
+  StopSDL()
+  restoreSDLConfigParameters()
+end
+
+--[[ @bytesToInt32: convert bytes to int32
+--! @parameters:
+--! val - value to convert
+--! offset - offset
+--! @return: value in int32
+--]]
+local function bytesToInt32(pVal, pOffset)
+  local res = bit32.lshift(string.byte(pVal, pOffset), 24) +
+  bit32.lshift(string.byte(pVal, pOffset + 1), 16) +
+  bit32.lshift(string.byte(pVal, pOffset + 2), 8) +
+  string.byte(pVal, pOffset + 3)
+  return res
+end
+
+--[[ @int32ToBytes: convert int32 to bytes
+--! @parameters:
+--! val - value to convert
+--! @return: value in bytes
+--]]
+local function int32ToBytes(pVal)
+  local res = string.char(
+    bit32.rshift(bit32.band(pVal, 0xff000000), 24),
+    bit32.rshift(bit32.band(pVal, 0xff0000), 16),
+    bit32.rshift(bit32.band(pVal, 0xff00), 8),
+    bit32.band(pVal, 0xff)
+  )
+  return res
+end
+
+--[[ @rpcPayload: create payload for RPC
+--! @parameters:
+--! msg - message table to populate
+--! @return: populated message
+--]]
+local function rpcPayload(pMsg)
+  pMsg.payload = pMsg.payload or ""
+  pMsg.binaryData = pMsg.binaryData or ""
+  local res = string.char(
+    bit32.lshift(pMsg.rpcType, 4) + bit32.band(bit32.rshift(pMsg.rpcFunctionId, 24), 0x0f),
+    bit32.rshift(bit32.band(pMsg.rpcFunctionId, 0xff0000), 16),
+    bit32.rshift(bit32.band(pMsg.rpcFunctionId, 0xff00), 8),
+    bit32.band(pMsg.rpcFunctionId, 0xff)) ..
+  int32ToBytes(pMsg.rpcCorrelationId) ..
+  int32ToBytes(#pMsg.payload) ..
+  pMsg.payload .. pMsg.binaryData
+
+  return res
+end
+
+--[[ @putFileByFrames: process PutFile RPC frame by frame
+--! @parameters:
+--! pParams - table with parameters (file, isSentDataEncrypted, isUnexpectedFrameInserted, isMalformedFrameInserted)
+--]]
+function m.putFileByFrames(pParams)
+  config.debug = true
```

Comment:
@dboltovskyi Please remove debug code and redundant whitespaces from this module.

Reply:
@aderiabin Accepted